packages/mui-base/src/SnackbarUnstyled/SnackbarUnstyled.types.ts

```diff
@@ -0,0 +1,70 @@
+import React from 'react';
+import { TransitionChildren } from 'react-transition-group/Transition';
+import { OverrideProps } from '@mui/types';
+import { ClickAwayListenerProps } from '../ClickAwayListener';
+import { UseSnackbarParameters } from './useSnackbar.types';
+import { SlotComponentProps } from '../utils';
+
+export interface SnackbarUnstyledComponentsPropsOverrides {}
+
+interface SnackbarUnstyledOwnProps extends Omit<UseSnackbarParameters, 'ref'> {
```

Comment:
Please export this interface - we've got reports saying not doing so causes issues.